.eslintrc

```diff
@@ -46,7 +46,7 @@
           "files": ["*.ts", "*.tsx"],
           "rules": {
               "@typescript-eslint/explicit-module-boundary-types": ["error"],
-              "@typescript-eslint/indent": ["error"],
```

Comment:
Intentional?

Reply:
Yeah, seems like this rule conflicts with prettier.

Reply:
Make sense!